pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1447,6 +1451,22 @@ func (c *AWSCloud) GetVolumeLabels(volumeName string) (map[string]string, error)
 	return labels, nil
 }
 
```

Comment:
My code reading says `getMountDevice()` gets the device path by querying compute instance's disk mapping info from cloud provider. It doesn't need to look at the local filesystem.